apps/web/lib/withEmbedSsr.tsx

```diff
@@ -15,10 +15,9 @@ export default function withEmbedSsr(getServerSideProps: GetServerSideProps) {
       const destinationUrlObj = new URL(ssrResponse.redirect.destination, "https://base");
 
       // Make sure that redirect happens to /embed page and pass on embed query param as is for preserving Cal JS API namespace
-      const newDestinationUrl = `${
+      const newDestinationUrl = `${destinationUrlObj.origin}${
```

Comment:
With the recent redirect being on another domain, it is important to use the origin as well otherwise it causes infinite redirect
User Migrated to Org Redirect(to org domain) -> This embed redirect (Stays on org domain) -> On Org domain, no redirect needed.